schemas/bolt-transport-definitions.json

```diff
@@ -199,6 +200,10 @@
       "type": "string",
       "enum": ["sh", "bash", "zsh", "dash", "ksh", "powershell"]
     },
+    "native-ssh": {
+      "description": "Whether to shell out directly to SSH or use the Ruby net-ssh library",
```

Comment:
```suggestion
      "description": "Whether to shell out directly to SSH or use the Ruby net-ssh library.",
```

Reply:
This was literally the first thing I did on this PR so I wouldn't forget

Reply:
Hopefully we can eventually have this generated so we can just forget about it :D